onnxruntime/contrib_ops/cuda/layer_norm_impl.h

```diff
@@ -36,8 +36,8 @@ void HostApplyLayerNorm(
     U* mean,
     U* invvar,
     const T* input,
-    int64_t n1,
-    int64_t n2,
+    int n1,
+    int n2,
```

Comment:
May be not a good change

Reply:
This is because the underlaying cuda implementation can only support int32.